ivy/functional/frontends/tensorflow/math.py

```diff
@@ -296,4 +296,8 @@ def zero_fraction(value, name="zero_fraction"):
     return ivy.divide(count_zero, ivy.add(count_zero, count_nonzero))
 
 
+def argmin(input, axis, output_type, name=None):
+    return ivy.argmin(input, axis=axis)
```

Comment:
The original function uses the `output_type` argument to cast the data type of the result before it's returned. We should also do this here.